ReadMe.md

```diff
@@ -1,7 +1,10 @@
 # Phantom
 ## Maintainability Badge
 <a href="https://codeclimate.com/github/atlp-rwanda/phantom-be-dream-team/maintainability"><img src="https://api.codeclimate.com/v1/badges/d9ba62bb03c827308555/maintainability" /></a>
+## CI/CD
+![Codecov](https://github.com/atlp-rwanda/phantom-be-dream-team/workflows/CI/CD/badge.svg)
 ## Coverage Badge
+![Codecov](https://img.shields.io/codecov/c/github/atlp-rwanda/phantom-be-dream-team/ch-setup-continous-continous-integration-#18)
```

Comment:
You are using only this branch in this coverage badge? It might cause the problem when we get to other branches. Check out Arthur's comment above.